docker-compose.test.yml

```diff
@@ -0,0 +1,22 @@
+version: '3.8'
```

Comment:
To run the integration tests, Postgres should be present locally.
Developers may have different ways to do that.

To simplify the project structure, remove it from the project tree, update the README, and explain that developers need to install Postgres themselves.

Reply:
@b00f I think Danh was trying to create an isolated environment for integration testing that met the requirements of the task or did we misunderstand you?

Reply:
> In the future, we may have multiple infra not only postgres. 

this is exactly why I insist not adding extra files for integration tests. Eventually, we end up with multiple docker-compose files (if we continue this approach) and differnt settings. Let's keep things simple.